src/status_im2/contexts/profile/create/events.cljs

```diff
@@ -1,21 +1,15 @@
 (ns status-im2.contexts.profile.create.events
   (:require
     [native-module.core :as native-module]
-    [re-frame.core :as re-frame]
     [status-im2.contexts.profile.config :as profile.config]
+    [status-im2.contexts.profile.create.effects :as effects]
     [utils.re-frame :as rf]
     [utils.security.core :as security]))
 
-(re-frame/reg-fx
- ::create-profile-and-login
- (fn [request]
-   ;;"node.login" signal will be triggered as a callback
-   (native-module/create-account-and-login request)))
-
 (rf/defn create-profile-and-login
   {:events [:profile.create/create-and-login]}
   [_ {:keys [display-name password image-path color]}]
-  {::create-profile-and-login
+  {::effects/create-profile-and-login
```

Comment:
we decided to do not use namespaced keywords

Reply:
I agree @flexsurfer, I just didn't want to rename anything from the original as I wrote in the description, just to play safe. But if you think it's worth in this PR I will double-check all namespaced keys I touched and rename them to something else using `:effects/`.

Reply:
yeah i think if we're doing this better to do it properly, i can help you with that

Reply:
I'll give it a try and let you know. It should be fine

Reply:
@flexsurfer, how would you name effects that are defined like:

```clojure
:contacts/resolve-public-key-from-ens
```

Would you rename them:

1. `:effects.contacts/resolve-public-key-from-ens`
2. `:effects/contacts.resolve-public-key-from-ens`
3. `:effects/contacts-resolve-public-key-from-ens`

I'm inclined for option 1 because it gives a clear separation between the domain and the actual effect that will be performed.

Reply:
although I've already done a few with 3, but 1 is better